lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -104,6 +108,62 @@ spec :: forall n t.
     , EncodeAddress n
     ) => SpecWith (Port "node", FeePolicy, Context t)
 spec = do
+    describe "HW_WALLETS_02,03 - Delegation with restored HW Wallets" $ do
```

Comment:
Should we move this module to core-integration and re-use it for shelley eventually?

Reply:
I think we will eventually once stake pools arrive. However there are some tests there that wait for the next epoch to make assertions, not sure if it can be done in a timely manner in shelley haskell node?

Reply:
👌